apps/passport-client/src/dispatch.ts

```diff
@@ -354,12 +354,14 @@ async function finishAccountCreation(
     return; // Don't save the bad identity.  User must reset account.
   }
 
-  // Save PCDs to E2EE storage.
+  // Save PCDs to E2EE storage.  knownRevision=undefined will overwrite
+  // any conflicting data which may already exist.
```

Comment:
worth commenting that it is quite unlikely that pre-existing data would exist given this is a new account w/ new encryption key